packages/component-library-css/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@gemeente-denhaag/component-library-css",
+  "version": "0.2.3-alpha.50",
```

Comment:
I'm guessing this package isn't starting on alpha version 50 ;o

Reply:
I make the same mistake all the time 🙈

Reply:
Whoops, is fixed now